udata/core/dataset/forms.py

```diff
@@ -60,6 +61,12 @@ class CommunityResourceForm(BaseResourceForm):
     organization = fields.PublishAsField(_('Publish as'))
 
 
+def map_legacy_frequencies(form, field):
```

Comment:
In that case, we should document it within the CHANGELOG and set an associated deprecation timeline somehow.